homeassistant/components/discord/notify.py

```diff
@@ -43,16 +48,21 @@ def file_exists(self, filename):
 
     async def async_send_message(self, message, **kwargs):
         """Login to Discord, send message to channel(s) and log out."""
-
         discord.VoiceClient.warn_nacl = False
         discord_bot = discord.Client()
         images = None
 
         if ATTR_TARGET not in kwargs:
             _LOGGER.error("No target specified")
             return None
+
         data = kwargs.get(ATTR_DATA) or {}
 
+        if ATTR_EMBED in data:
+            embedding = data.get(ATTR_EMBED)
```

Comment:
```suggestion
            embedding = data[ATTR_EMBED]
```
With the line directly above it, you know that it's there.